Client/Ecosia/UI/NTP/OnboardingCard/NTPOnboardingCardCellViewModel.swift

```diff
@@ -0,0 +1,77 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Shared
+import Core
+import Common
+
+protocol NTPOnboardingCardCellDelegate: AnyObject {
+    func onboardingCardClick()
+    func onboardingCardDismiss()
+}
+
+final class NTPOnboardingCardCellViewModel {
+    weak var delegate: NTPOnboardingCardCellDelegate?
+    var theme: Theme
+    
+    init(delegate: NTPOnboardingCardCellDelegate? = nil, theme: Theme) {
+        self.delegate = delegate
+        self.theme = theme
+    }
+}
+
+extension NTPOnboardingCardCellViewModel: HomepageViewModelProtocol {
+    func setTheme(theme: Theme) {
+        self.theme = theme
+    }
+
+    var sectionType: HomepageSectionType {
+        return .onboardingCard
+    }
+
+    var headerViewModel: LabelButtonHeaderViewModel {
+        return .emptyHeader
+    }
+
+    func section(for traitCollection: UITraitCollection, size: CGSize) -> NSCollectionLayoutSection {
+        let itemSize = NSCollectionLayoutSize(widthDimension: .fractionalWidth(1.0),
+                                              heightDimension: .fractionalHeight(1.0))
+        let item = NSCollectionLayoutItem(layoutSize: itemSize)
+
+        let groupSize = NSCollectionLayoutSize(widthDimension: .fractionalWidth(1.0),
+                                               heightDimension: .estimated(128)) // TODO: Make cell automatically size it's height
+        let group = NSCollectionLayoutGroup.horizontal(layoutSize: groupSize, subitem: item, count: 1)
+
+        let section = NSCollectionLayoutSection(group: group)
+
+        section.contentInsets = sectionType.sectionInsets(traitCollection, bottomSpacing: 8)
+
+        return section
+    }
+
+    func numberOfItemsInSection() -> Int {
+        return 1
+    }
+
+    var isEnabled: Bool {
+//        OnboardingCardNTPExperiment.shouldShowCard
+        true // TODO: Why is there a `UICollectionViewCompositionalLayout` whenever this is false?
```

Comment:
Getting a strange error whenever this is false which seems to be related to the `UICollectionViewCompositionalLayout`. Cannot reproduce this on other similar cells like `NTPBookmarkNudgeCellViewModel` and couldn't find differences in setup 🤔 

![Screenshot 2024-09-09 at 17 25 41](https://github.com/user-attachments/assets/c196e157-6615-47db-aa92-8a01048934b7)

Reply:
Solved ✅ . The root cause was essentially the order by which the `viewModel`s in the `HomepageViewController` where defined. Added a supporting comment as well.